cipd_packages/codesign/test/file_codesign_visitor_test.dart

```diff
@@ -944,46 +950,38 @@ status: Invalid''',
           .toList();
       expect(
         messages,
-        contains('Failed to upload to the notary service with args: '
-            'xcrun notarytool submit ${rootDirectory.absolute.path}/temp '
-            '--apple-id <appleID-redacted> --password <appSpecificPassword-redacted> --team-id <teamID-redacted> '
-            '--verbose'),
-      );
-      expect(
-        messages,
-        contains('Trying again 2 more times...'),
+        contains('Failed to upload to the notary service'),
       );
     });
 
     test('upload notary throws exception if exit code is unnormal', () async {
       fileSystem.file('${rootDirectory.absolute.path}/temp').createSync();
-      processManager.addCommands(<FakeCommand>[
-        FakeCommand(
-          command: <String>[
-            'xcrun',
-            'notarytool',
-            'submit',
-            '${rootDirectory.absolute.path}/temp',
-            '--apple-id',
-            fakeAppleID,
-            '--password',
-            fakePassword,
-            '--team-id',
-            fakeTeamID,
-            '--verbose',
-          ],
-          stdout: '''Error uploading file.
+      for (int i = 0; i < 3; i++) {
```

Comment:
umm should this be 5 attempts if we update max attempts from 3 to 5. 

nit: I guess the test always pass whether the input is 3 or 5. maybe checking the number of repetitions in records would catch this but its nit.

Reply:
Generally, refactors should avoid changing the tests too much. In this case, I kept it as 3 instead of 5 as that's probably a good low number to verify the retries pass through exceptions.